[2025-04-23T09:40:55.812+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-23T09:40:55.823+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_data_daily.load_backfill_data_2 manual__2025-04-23T09:40:46.395557+00:00 [queued]>
[2025-04-23T09:40:55.826+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_data_daily.load_backfill_data_2 manual__2025-04-23T09:40:46.395557+00:00 [queued]>
[2025-04-23T09:40:55.827+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-23T09:40:55.832+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): load_backfill_data_2> on 2025-04-23 09:40:46.395557+00:00
[2025-04-23T09:40:55.838+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6592) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-23T09:40:55.839+0000] {standard_task_runner.py:72} INFO - Started process 6594 to run task
[2025-04-23T09:40:55.840+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_weather_data_daily', 'load_backfill_data_2', 'manual__2025-04-23T09:40:46.395557+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/el_daily.py', '--cfg-path', '/tmp/tmpez2_zbsv']
[2025-04-23T09:40:55.841+0000] {standard_task_runner.py:105} INFO - Job 251: Subtask load_backfill_data_2
[2025-04-23T09:40:55.863+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_weather_data_daily.load_backfill_data_2 manual__2025-04-23T09:40:46.395557+00:00 [running]> on host eb276ba0497d
[2025-04-23T09:40:55.951+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_weather_data_daily' AIRFLOW_CTX_TASK_ID='load_backfill_data_2' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T09:40:46.395557+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T09:40:46.395557+00:00'
[2025-04-23T09:40:55.952+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-23T09:40:55.966+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-23T09:40:55.971+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-23T09:40:55.971+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fetch_weather_data_daily, task_id=load_backfill_data_2, run_id=manual__2025-04-23T09:40:46.395557+00:00, execution_date=20250423T094046, start_date=20250423T094055, end_date=20250423T094055
[2025-04-23T09:40:56.012+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-23T09:40:56.026+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-23T09:40:56.027+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
